testsuite/preview/pom.xml

```diff
@@ -50,12 +50,274 @@
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.resource</groupId>
+                <artifactId>jakarta.resource-api</artifactId>
+                <version>${preview.version.jakarta.resource.jakarta-resource-api}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.jws</groupId>
+                <artifactId>jakarta.jws-api</artifactId>
+                <version>${preview.version.jakarta.jws.jakarta-jws-api}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.ws.rs</groupId>
+                <artifactId>jakarta.ws.rs-api</artifactId>
+                <version>${preview.version.jakarta.ws.rs.jakarta-ws-rs-api}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
+                <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
+                <version>${preview.version.org.jboss.spec.jakarta.xml.ws.api_3.0_spec}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.wildfly</groupId>
+                <artifactId>wildfly-connector-jakarta</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            
+            <!-- Temporary CXF References -->
+            <dependency>
+                <groupId>org.jboss.ws.cxf</groupId>
+                <artifactId>jbossws-cxf-client-ee9</artifactId>
+                <version>${preview.version.org.jboss.ws.cxf}</version>
+                <scope>system</scope>
+                <systemPath>${wildfly.transformed.repo.dir}/org/jboss/ws/cxf/jbossws-cxf-client/${preview.version.org.jboss.ws.cxf}/jbossws-cxf-client-${preview.version.org.jboss.ws.cxf}.jar</systemPath>
+            </dependency>
```

Comment:
@darranl I think L158-165 is the only thing that should be added to dependencyManagement in this pom. L52  already results in importing the wildfly-ee9-parent bom, and then all the other entries are duplicates because that bom declares them.